### PR TITLE
INFENG-11424 Release v3.9.5

### PR DESCRIPTION
**Pull request by bender-wk**
JIRA: https://jira.webfilings.com/browse/INFENG-11424

@Workiva/product2

* #1263 INFENG-11402 remove codecov
* #1265 SAPI-1500: Update testing
* #1266 INFENG-11460: Move mockito to dev dependency
* #1257 INFENG-11295 Java: Bump checkstyle from 8.30 to 8.31 in /lib/java
* #1259 java: Add TTransportExceptionType.DISCONNECTED
* #1255 INFENG-11293 Tests: Bump jetty-servlet from 9.4.27.v20200227 to 9.4.28.v20200408 in /test/integration/java/frugal-integration-test
* #1268 INFENG-11492 Examples: Bump netty-all from 4.1.48.Final to 4.1.49.Final in /examples/java
* #1270 INFENG-11494 Tests: Bump netty-all from 4.1.48.Final to 4.1.49.Final in /test/integration/java/frugal-integration-test
* #1271 INFENG-11495 Java: Bump netty-all from 4.1.48.Final to 4.1.49.Final in /lib/java
* #1269 INFENG-11493 Java: Bump checkstyle from 8.31 to 8.32 in /lib/java
* #1256 INFENG-11294 Java: Bump jnats from 2.6.6 to 2.6.7 in /lib/java
* #1273 INFENG-11560 Go: Bump github.com/go-stomp/stomp from 2.0.5+incompatible to 2.0.6+incompatible in /lib/go
* #1274 INFENG-11561 Python: Bump webtest from 2.0.34 to 2.0.35 in /lib/python
* #1272 INFENG-11584: java: Support custom ExecutorService on a per-FServlet basis

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
<details>
<summary>(3) Vulnerable direct dependencies were detected</summary>
<li>2 vulns in <code>org.apache.thrift:libthrift < 0.12.0</code> via <code>examples/java/pom.xml</code></li>
<li>2 vulns in <code>org.apache.thrift:libthrift < 0.12.0</code> via <code>lib/java/pom.xml</code></li>
<li>2 vulns in <code>org.apache.thrift:libthrift < 0.12.0</code> via <code>test/integration/java/frugal-integration-test/pom.xml</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=frugal">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: test/'../test/integration/dart/gen-dart'/frugal_test/lib/frugal_test.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Autogenerated by Frugal Compiler (3.9.5)
```
Comment (yuanmwang-wf) [inline test/'../test/integration/dart/gen-dart'/frugal_test/lib/frugal_test.dart:1]: This file path looks wrong

File: test/'../test/integration/dart/gen-dart'/frugal_test/lib/src/f_bonk.dart [hunk 2]
```diff
@@ -0,0 +1,200 @@
+// Autogenerated by Frugal Compiler (3.9.5)
```
Comment (yuanmwang-wf) [inline test/'../test/integration/dart/gen-dart'/frugal_test/lib/src/f_bonk.dart:1]: This file path looks wrong

File: test/'../test/integration/dart/gen-dart'/frugal_test/lib/src/f_event.dart [hunk 3]
```diff
@@ -0,0 +1,208 @@
+// Autogenerated by Frugal Compiler (3.9.5)
```
Comment (yuanmwang-wf) [inline test/'../test/integration/dart/gen-dart'/frugal_test/lib/src/f_event.dart:1]: This file path looks wrong

**Pull request closed without merging**